package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.0.0-development",
   "main": "lib/BrandwatchReactAuth.cjs.js",
   "module": "lib/BrandwatchReactAuth.esm.js",
-  "license": "MIT",
```

Comment:
I believe this makes the package.json invalid, if you use `"UNLICENSED"` as the license it retains full copyright over the source.